app/templates/components/warning-message.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="warning">
+    <img src="assets/images/warning.png" alt="warning icon">
+    <p>Render times may be inaccurate due to instrumentation and environment</p>
+    <button {{ action closeWarning }}> close </button>
```

Comment:
Can you remove spaces at the beginning and end? `{{action closeWarning}}`. Also the action name is fine but I would've named it just `close` given that we are inside the warning message :)